Content.Server/Atmos/EntitySystems/GasTankSystem.cs

```diff
@@ -51,6 +55,12 @@ public override void Initialize()
             SubscribeLocalEvent<GasTankComponent, GasAnalyzerScanEvent>(OnAnalyzed);
             SubscribeLocalEvent<GasTankComponent, PriceCalculationEvent>(OnGasTankPrice);
             SubscribeLocalEvent<GasTankComponent, GetVerbsEvent<AlternativeVerb>>(OnGetAlternativeVerb);
+            Subs.CVar(_cfg, CCVars.AtmosTankFragment, UpdateMaxRange);
```

Comment:
```suggestion
            Subs.CVar(_cfg, CCVars.AtmosTankFragment, UpdateMaxRange, true);
```
This needs the `invokeImmediately` argument set to true in order to correctly update at the beginning.